Gopkg.lock

```diff
@@ -1184,6 +1189,7 @@
     "pkg/util"
   ]
   revision = "6702109cc68eb6fe6350b83e14407c8d7309fd1a"
+  version = "kubernetes-1.11.0"
```

Comment:
There are some changes in here I wasn't expecting, but I guess that is just due to a newer point release of Dep you are using?  ie there are some differences in version and branch elements.

Reply:
This has to do with the information available in git at the time we last edited the lock file. When someone updated us to 1.11, they pointed at the branch, not the tag I'm assuming.

This won't change the commit used in our vendor directory, just tweaks the breadcrumb trail on how to find that SHA (i.e. prefer the tag over a moving branch).